test/test_translator.py

```diff
@@ -84,27 +81,29 @@ def test_parse_incident() :
     ]
   }
 }
+    assert test_feature == valid_feature
 
+def test_parse_incident_success_no_data():
```

Comment:
Since we're not testing a "successful" case here can we name it something more appropriate? Maybe `test_parse_incident_no_data` where we just drop the "success" text as it's missleading.